### ISSUE TITLE
0.045tr - RygenB Point Blank Vanar

### ISSUE THREAD

**Issue by Sparen** [bluebear94/cy05#4]
As is, RygenB is currently capable of point-blank one-shitting all of Vanar's non spells and some of her spells.

Suggestion:

```
task SetDamageRateAtFrame(obj, shot,bomb,frame){//Talos
  ObjEnemy_SetDamageRate(obj, 0,0);
  let time=frame;
  while(IsPlayerSpellActive){ //Prevents Bomb Damage from previous attack. 
    time--; //However, makes stage enemies immune to bombs because it just yields after time hits 0.
    yield;
  }
  while(time>0){
    time--;
    yield;
  }
  ObjEnemy_SetDamageRate(obj,shot,bomb);
}
```

**Comment by bluebear94** [comment]
What if I change lines 33 - 35 to:

```
task TCutIn {
    while (GetPlayerInvincibilityFrame > 0 || IsPlayerSpellActive) {yield;}
    ObjEnemy_SetDamageRate(objBoss, 100, bombDamageRates[level - 1]);
```

**Comment by Sparen** [comment]
As long as you set the damage rate to (0,0) beforehand, it should be fine. I don't understand what level is, so I can't comment on that section

**Comment by bluebear94** [comment]
Fixed in 84b84eeeec15d736297956d4b1b6033a1d672312.
